slf4j/src/main/scala/zio/logging/slf4j/Slf4jLogger.scala

```diff
@@ -53,4 +54,40 @@ object Slf4jLogger {
       },
       rootLoggerName = rootLoggerName
     )
+
+  def makeWithAnnotationsAsMdc(
```

Comment:
please add example to documentation and test if possible

Reply:
@pshemass sure, will do that 👍

Reply:
@pierangeloc any update on that? do you need help?

Reply:
@pshemass sorry I got dragged by other tasks during the week. I'm done with testing, need just to update docs. I should be done with that today/tomorrow

Reply:
@pshemass added test and docs, can you have a look?